src/lib/registration/ui/addon_selection_base_dialog.rb

```diff
@@ -92,9 +92,9 @@ def content
         vbox_elements = [Left(Heading(heading))]
         available_addons = @all_addons.reject(&:registered?)
 
-        unless available_addons.empty? || available_addons.select(&:beta_release?).empty?
+        unless available_addons.empty? || available_addons.all?(&:supported_release?)
           vbox_elements.push(Left(CheckBox(Id(:filter_beta), Opt(:notify),
-            _("&Hide Beta Versions"), check_filter)))
+            _("&Hide Development Versions"), check_filter)))
```

Comment:
- Maybe  "&Hide Testing Versions" would be better, for customers they are ment for testing, development is rather for us (SUSE).
- Add `# TRANSLATORS: Checkbox label, hides alpha or beta versions (not released yet)` comment so the meaning is clear for the translators.